src/Core/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommand.cs

```diff
@@ -74,6 +74,15 @@ public async Task UpdateSecretsManagerSubscription(SecretsManagerSubscriptionUpd
         await SendEmailIfAutoscaleLimitReached(organization);
     }
 
+    public async Task AdjustSecretsManagerServiceAccountsAsync(Guid organizationId, int smServiceAccountsAdjustment)
+    {
+        var organization = await _organizationRepository.GetByIdAsync(organizationId);
```

Comment:
We should avoid database reads in commands. Have the caller pass in the `organization` instead.